.github/workflows/ci-cd.yml

```
@@ -208,7 +208,7 @@ jobs:
     - name: Get pip cache dir
       id: pip-cache
       run: |
-        echo "::set-output name=dir::$(pip cache dir)"    # - name: Cache
+        echo "dir=$(pip cache dir)" >> $GITHUB_OUTPUT
```

This is my preferred style in other projects. Let's match it:
```suggestion
        echo "dir=$(pip cache dir)" >> "${GITHUB_OUTPUT}"
```

---

Ah, I've missed curly braces 🤦
Sorry about that.